### ISSUE TITLE
Eliminate dependency on angular-visjs

### ISSUE THREAD

**Issue by grtjn** [patrickmcelwee/ml-visjs-graph-ng#25]
Angular-visjs does make integration in angular easier, but it makes it more difficult to provide framework-agnostic support. Since it is pretty thin anyhow, I think we can copy over essential parts, and drop the dependency pretty easily. After that, we can try to make the core work with plain JS as much as possible.

**Comment by patrickmcelwee** [comment]
Yeah, or extract as much of the framework-agnostic stuff into a separate library, which this one depends on. And this essentially becomes the angular wrapper.
